src/scripts/accesskit.css

```diff
@@ -37,3 +37,24 @@ body.accesskit-blue-links article a[target="_blank"][href^="https://href.li/"] {
 body.accesskit-no-user-colours article span[style^="color"] {
   color: inherit !important;
 }
+
+
+figure.accesskit-visible-alt-text {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+figure.accesskit-visible-alt-text figcaption {
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 2px;
+  padding: 5px;
+
+  line-height: 1;
+  text-align: center;
+  background: #ddd;
+  color: #111;
```

Comment:
Rule no.1 of inserting elements into the webpage: remember that different palettes exist!

I get that it's not easy to figure out how to use all the different variables, though. Personally, for this sort of complimentary in-post element, I would use "black" text on a "secondary accent" background.

| True Blue | Dark Mode | Low-Contrast Classic |
|-----------|------------|----------------------|
| ![Screen Shot 2021-07-29 at 14 57 21](https://user-images.githubusercontent.com/28949509/127505429-e9ee38e9-dbd3-4043-81a0-05df570e9acf.png) | ![Screen Shot 2021-07-29 at 14 57 35](https://user-images.githubusercontent.com/28949509/127505458-ac21cf99-deb5-4cdc-97cc-77500ddf6493.png) | ![Screen Shot 2021-07-29 at 14 59 29](https://user-images.githubusercontent.com/28949509/127505494-1c1be5b3-2638-49fe-9c1d-d6672e3a58ad.png) |



```suggestion
  background-color: rgb(var(--secondary-accent));
  color: rgb(var(--black));
```

Reply:
LOL, here I was wondering if there was some magical way to detect the current theme colors, and the variables were right in front of me the whole time... noob mistakes 😆